test/db.js

```diff
@@ -0,0 +1,10 @@
+const models = require('../models');
+
+module.exports = {
+    async deleteAllRows() {
```

Comment:
Not a blocker, but could we run these in parallel?

Reply:
I believe the first three can, `Accounts` has to be deleted last because there are foreign keys referencing `id`